kms/snippets/snippet_test.go

```diff
@@ -25,7 +25,7 @@ func init() {
 	_ = google.DefaultClient
 }
 
-func createKeyring(project, name string) error {
+func createKeyring(project, keyRing string) error {
```

Comment:
Gotta make this change in `sample-template.go` too, then run `go generate` from the `snippets directory.

Reply:
Thanks, I tried running `go generate` from the `snippets/` and nothing change.